soc/nordic/nrf54h/Kconfig

```
@@ -53,3 +53,8 @@ config SOC_NRF54H20_CPUPPR
 
 config SOC_NRF54H20_CPUFLPR
 	depends on RISCV_CORE_NORDIC_VPR
+
+config SOC_NRF54H20_NO_MRAM_LATENCY
+	bool "Disallow MRAM latency"
+	depends on NRFS && NRFS_MRAM_SERVICE_ENABLED
```

Would it be a good idea to change this to:
```suggestion
	imply NRFS
	imply NRFS_MRAM_SERVICE_ENABLED
```
if we can avoid changing the defconfig?

---

right, will change